bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/RetryOptions.java

```diff
@@ -54,7 +54,8 @@
       Status.Code.DEADLINE_EXCEEDED,
       Status.Code.INTERNAL,
       Status.Code.UNAVAILABLE,
-      Status.Code.ABORTED);
+      Status.Code.ABORTED,
```

Comment:
I think keeping retries in the retry listener makes this code a lot easier to reason about, especially with the addition of serverside token revocations. I don't think having 2 separate sets of retry loops adds a lot of value.